openwisp_radius/templates/radius_accounting_start.html

```diff
@@ -2,4 +2,5 @@
 <div class="msg">
 <p>{% trans "A new session has been started for your account:" %} {{ user.get_username }}.</p>
 <p>{% trans "You can review your session to find out how much time and/or traffic has been used or you can terminate the session at any time by clicking on the button below." %}</p>
+{% if sesame_max_age %}<p>{% blocktrans with sesame_max_age|timesince as sesame_max_age %}NOTE: This link is valid only for {{sesame_max_age}}{% endblocktrans %}.</p>{% endif %}
```

Comment:
I think using [`naturaltime`](https://docs.djangoproject.com/en/4.0/ref/contrib/humanize/#naturaltime) will be better here. Since, it renders time as `an hour from now.`